docs/privacy-policy.md

```diff
@@ -0,0 +1,33 @@
+# Privacy Policy
+
+*Effective 2021-03-22*
+
+## Collection of Information
+
+### Information you give
+Buzz About receives personal information when you sign in. This may include your email address and your phone number.
+
+### Information we collect automatically
+We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your machine, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.
```

Comment:
```suggestion
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your computer or device, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.
```

Reply:
```suggestion
We collect information about your interactions in Buzz About. While Buzz About itself does not collect any information automatically about your device, third party services that we use may collect that information. Check the privacy policies of our third party services below to read about the information they collect.
```

I would prefer just saying "device". Good catch KK, something like "device" is far more common for this type of writing than "machine"